targets/csharp/source/source/PlayFabHttp/PlayFabSysHttp.cs

```diff
@@ -12,6 +11,14 @@ namespace PlayFab.Internal
 {
     public class PlayFabSysHttp : IPlayFabHttp
     {
+		
+		private readonly HttpClient _client;
+
+        public PlayFabSysHttp()
+        {
+            _client = new HttpClient();
```

Comment:
Constructor not necessary. You can set this on line 15

Reply:
I'll fix that. The point was that the next person to modify HttpClient settings won't have to do this small refactoring.

Reply:
fixed